Readme.md

```diff
@@ -359,10 +366,15 @@ Closes the currently opened output file.
 .orga FileAddress
 ```
 
-Sets the output pointer to the specified address. ```.org``` specifies a memory address, which is automatically converted to the file address for the current output file. ```.orga``` directly specifies the file address.
+Sets the output pointer to the specified address. `.org` specifies a memory address, which is automatically converted to the file address for the current output file. `.orga` directly specifies the absolute file address.
 
 ### Changing the header size
 
+```
+.headersize Offset
+```
+
+Sets the header size to the given value which is difference between the first byte of the file and its position in memory. This is used to calculate all addresses up until the next .headersize or .open/.create directive. The current address will also be set to `Offset`.
```

Comment:
> The current address will also be set to `Offset`

is outdated, now the file address and position is preserved while the memory address is recalculated according to the new header size. Also a good idea to mention that negative header sizes are now supported as long as the new memory address remains positive.